frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -568,9 +568,14 @@ function ReaderFooter:onPageUpdate(pageno)
     self:updateFooterPage()
 end
 
-function ReaderFooter:onPosUpdate(pos)
+function ReaderFooter:onPosUpdate(pos, pageno)
     self.position = pos
     self.doc_height = self.view.document.info.doc_height
+    if pageno then
+        self.pageno = pageno
+        self.pages = self.view.document:getPageCount()
+        self.ui.doc_settings:saveSetting("doc_pages", self.pages) -- for Book information
```

Comment:
Do we need save doc_pages every time when we change book position?

Reply:
`saveSettings()` just updates the value of `doc_settings["doc_pages"]`, it does not save/flush/write do disk, so it costs nearly nothing. And it is there in onPageUpdate just above this onPosUpdate :)

Reply:
Thank. I thought that this line write to disk :)